docs/reference/gpu_arch.md

```diff
@@ -17,3 +17,17 @@
 - [AMD CDNA Architecture Whitepaper](https://www.amd.com/system/files/documents/amd-cdna-whitepaper.pdf)
 - [AMD Vega Architecture Whitepaper](https://en.wikichip.org/w/images/a/a1/vega-whitepaper.pdf)
 - [AMD RDNA Architecture Whitepaper](https://www.amd.com/system/files/documents/rdna-whitepaper.pdf)
+
+## Architecture Guides
+
+:::::{grid} 1 1 1 1
+:gutter: 1
+
+:::{grid-item-card} AMD Instinct MI200
+This chapter briefly reviews hardware aspects of the AMD Instinct MI250 accelerators and the CDNA™ 2 architecture that is the foundation of these GPUs.
+
+- [Instruction Set Architecture](https://www.amd.com/system/files/TechDocs/instinct-mi200-cdna2-instruction-set-architecture.pdf)
+- [Whitepaper](https://www.amd.com/system/files/documents/amd-cdna2-white-paper.pdf)
```

Comment:
Considering recent... events, I wonder if we should consider including these locally, in the repository, as non-code/non-documentation binary artifacts (i.e. under `/docs/data/foo`), just like the images? Opinions, @saadrahim ?

Reply:
I am not sure if we want it locally. If the binary artifacts are already stored on an AMD site, isn't it better to host it there? I understand links maybe broken by third parties, I am not sure if growing this repo size is the correct response.